### PR TITLE
multiplication table

### PR DESCRIPTION
**Pull request by dassmruti**

### CODE REVIEW

File: evenodd.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Evenodd
+  def self.check(num)
+    if(num%2==0)
+    puts "  "+num.to_s+ "is even"+"\n"
```
Comment (santosh-1987) [inline evenodd.rb]: Use return statement instead of puts, Try this :

return "#{num} is even."

File: evenodd.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    else
+    puts "  "+num.to_s+ " is odd"+"\n"
```
Comment (santosh-1987) [inline evenodd.rb]: return "#{num} is odd."

**Review by santosh-1987 — CHANGES_REQUESTED** [review]
Try to use return statements from a method .

File: evenodd.rb (hunk 1 above)
Comment (dassmruti) [reply evenodd.rb]: ok , thanks

**Comment by santosh-1987** [comment]
:shipit:

**Review by santosh-1987 — APPROVED** [review]

**Pull request merged**